docs/guide/server.md

```diff
@@ -0,0 +1,122 @@
+# Using Dolos Server
+
+We provide a free online service to run Dolos directly in our browser.
+You can use this service at [dolos.ugent.be/server](https://dolos.ugent.be/server).
+
+If you want to follow along, you can download a [ZIP with sample files](/simple-dataset.zip) which you can analyze on our server.
+
+## Upload and analyze files
+
+![Screenshot of the upload form](/images/screenshots/dolos-upload.png)
+
+To start a plagiarism analysis, upload a ZIP-file containing the files you want to analyze.
+You can specify a name for your analysis, which will be used to identify it in the report history.
+
+Additionally, you can select the programming language of the files.
+Dolos will automatically detect the language based on the file extension, but you can override this if needed.
+
+Click "Analyze" to upload your files and start the analysis.
+
+
+### Upload format
+
+Currently, we only support uploading a single ZIP-file containing all the files you want to analyze with an required `info.csv` file.
```

Comment:
Why do we require an info.csv file? Can't we run without it and just take all files inside the zip? Creating such file might be a barrier for others wanting to try dolos. If an info file is absent, we could show a hint on the result page that adding one with additional metadata might improve the results.

Reply:
I think the initial reason was because there were some problems with automatic programming language detection in that case.

But that does not seem the case anymore and should be straightforward to implement.